source/rails-girls/_js.haml

```diff
@@ -0,0 +1,13 @@
+:javascript
+  !function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
+
+
+.fb-root
+  :javascript
+    (function(d, s, id) {
+    var js, fjs = d.getElementsByTagName(s)[0];
+    if (d.getElementById(id)) return;
+    js = d.createElement(s); js.id = id;
+    js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.6&appId=603249739823141";
+    fjs.parentNode.insertBefore(js, fjs);
+    }(document, 'script', 'facebook-jssdk'));
```

Comment:
I strongly disagree with the inclusion of these social buttons scripts.

They are known to be statistically ineffective, kill performance (especially on mobile), and have (most important for me) big privacy issues.

An alternative, if we really want to include social buttons, is to use script-less ones, i.e. simple URLs pointing to the social site HTTP API. I (or Boris) can provide examples if needed.